bitmap-destruction-force/lib/bitmapFileHelper.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+var buf;
```

Comment:
It's unnecessary to create this variable; just pass data directly into your callbacks :)